source/vibe/data/json.d

```diff
@@ -1045,11 +1052,10 @@ struct Json {
 
 	private void initBigInt()
 	{
-		// BigInt is a struct, and it have a special BigInt.init value,  differs the null.
-		// m_data has no special initializer and when we tries to first access to BigInt
+		// BigInt is a struct, and it has a special BigInt.init value, which differs from null.
+		// m_data has no special initializer and when it tries to first access to BigInt
 		// via m_bigInt(), we should explicitly initialize m_data with BigInt.init
-		BigInt init_;
-		(cast(ubyte*)m_data.ptr)[0 .. BigInt.sizeof] = (cast(ubyte*)&init_)[0 .. BigInt.sizeof];
+		(cast(BigInt[1])m_data[0 .. BigInt.sizeof])[0] = BigInt.init;
```

Comment:
Just noticed this now - this is fine as long as `BigInt` has no postblit constructor or an assignment operator, but will break if it ever switches to reference counting for example. Does `m_data[0 .. BigInt.sizeof] = cast(void[])init_;` with `BigInt[1] init_;` work?